packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -205,6 +205,24 @@
     "primaryButtonText": "Continue without microphone",
     "ariaLabel": "Allow microphone access"
   },
+  "CameraAndMicrophoneDomainPermissionsDenied": {
+    "primaryText": "Unable to access camera and microphone",
+    "secondaryText": "Click the lock icon in the address bar to grant microphone permissions to this webpage. A page refresh may be required.",
+    "primaryButtonText": "Continue Anyway",
```

Comment:
nit: "Continue Anyway" => "Continue anyway"

Reply:
Oh good catch, this should actually be `Continue without x/y` - copy paste mistake!

Reply:
updated